docfx_project/index.md

```diff
@@ -1,117 +1,67 @@
-contextOptions.Geolocation = new Geolocation { Longitude = 12.492507m, Latitude = 41.889938m };
-contextOptions.Permissions = new[] { ContextPermission.Geolocation };
-
-var context = await browser.NewContextAsync(contextOptions);
-var page = await context.NewPageAsync();
-await page.GoToAsync("https://maps.google.com");
-page.ClickAsync("text='Your location'"); //
-await page.ScreenshotAsync("colosseum-iphone.png");
-```
-
-## Evaluate in browser context
-This code snippet navigates to example.com in Firefox, and executes a script in the page context.
-
-```cs
-await Playwright.InstallAsync();
-using var playwright = await Playwright.CreateAsync();
-await using var browser = await playwright.Firefox.LaunchAsync();
-
-var context = await browser.NewContextAsync();
-var page = await context.NewPageAsync();
-await page.GoToAsync("https://www.example.com/");
-var dimensions = await page.EvaluateAsync<Size>(@"() => {
-    return {
-        width: document.documentElement.clientWidth,
-        height: document.documentElement.clientHeight,
-    }
-}");
-Console.WriteLine(dimensions);
-```
-
-## Intercept network requests
-This code snippet sets up request routing for a WebKit page to log all network requests.
-
-```cs 
-await Playwright.InstallAsync();
-using var playwright = await Playwright.CreateAsync();
-await using var browser = await playwright.Firefox.LaunchAsync();
-
-var context = await browser.NewContextAsync();
-var page = await context.NewPageAsync();
-// Log and continue all network requests
-await page.RouteAsync("**", (route, _) =>
-{
-    Console.WriteLine(route.Request.Url);
-    route.ContinueAsync();
-});
-
-await page.GoToAsync("http://todomvc.com");
-```
-
-# Monthly reports
- * [October 2020](https://www.hardkoded.com/blog/playwright-sharp-monthly-oct-2020)
- * [September 2020](https://www.hardkoded.com/blog/playwright-sharp-monthly-sep-2020)
- * [July 2020](https://www.hardkoded.com/blog/playwright-sharp-monthly-jul-2020)
- * [June 2020](https://www.hardkoded.com/blog/playwright-sharp-monthly-jun-2020)
- * [May 2020](https://www.hardkoded.com/blog/playwright-sharp-monthly-may-2020)
- * [April 2020](https://www.hardkoded.com/blog/playwright-sharp-monthly-apr-2020)
- * [March 2020](https://www.hardkoded.com/blog/playwright-sharp-monthly-march-2020)
-
-# Useful links
-
-* [StackOverflow](https://stackoverflow.com/search?q=playwright-sharp)
-* [Issues](https://github.com/microsoft/playwright-sharp/issues?utf8=%E2%9C%93&q=is%3Aissue)
+---
+title: Playwright Sharp
+documentType: index
+---
+<style type="text/css">
+footer{
+  position: relative;
+}
+</style>
+
+<div class="hero">
+  <div class="wrap">
+    <div class="text">
+      <strong>Playwright Sharp</strong>
+    </div>
+    <div class="buttons-unit-small">
+      <a class="version-link" href="https://github.com/microsoft/playwright-sharp/releases">Release Notes</a><span>|</span><a class="github-link" href="https://github.com/microsoft/playwright-sharp">View on Github</a>
```

Comment:
```suggestion
      <a class="version-link" href="https://github.com/microsoft/playwright-sharp/releases">Release Notes</a><span>|</span><a class="github-link" href="https://github.com/microsoft/playwright-sharp">View on GitHub</a>
```

I know a bunch of employees from GitHub and we take the right capitalization very seriously.

Reply:
Too late, PRs accepted 😂